Mlem/App/Views/Pages/Community/CommunityView.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  CommunityView.swift
+//  Mlem
+//
+//  Created by Sjmarf on 30/07/2024.
+//
+
+import LemmyMarkdownUI
+import MlemMiddleware
+import SwiftUI
+
+struct CommunityView: View {
+    enum Tab: String, CaseIterable, Identifiable {
+        case posts, comments, about, moderation, details
+
+        var id: Self { self }
+        var label: LocalizedStringResource {
+            switch self {
+            case .posts: "Posts"
+            case .comments: "Comments"
+            case .about: "About"
+            case .moderation: "Moderation"
+            case .details: "Details"
+            }
+        }
+    }
+        
+    @Environment(AppState.self) var appState
+    @Environment(NavigationLayer.self) var navigation
+    @Environment(Palette.self) var palette
+    
+    @State var community: AnyCommunity
+    @State private var selectedTab: Tab = .posts
+    @State var showRefreshPopup: Bool = false
+    @State var postFeedLoader: CommunityPostFeedLoader?
+    
+    init(community: AnyCommunity) {
+        self.community = community
+    }
+    
+    var body: some View {
+        ContentLoader(model: community) { proxy in
+            if let community = proxy.entity, let postFeedLoader {
+                content(community: community, postFeedLoader: postFeedLoader)
+                    .refreshable {
+                        showRefreshPopup = false
+                        do {
+                            try await postFeedLoader.refresh(clearBeforeRefresh: false)
+                        } catch {
+                            handleError(error)
+                        }
+                    }
```

Comment:
Maybe bikeshedding, but we duplicate this same logic in `FeedsView` (with a little extra to pick the loader)--we maybe want to unify it into a `.refreshFeed` extension similar to `.loadFeed`. That would require some minor reworking of `FeedsView` to fit the canonical pattern.

Reply:
Good call 👍 Requires [this MlemMiddleware PR](https://github.com/mlemgroup/MlemMiddleware/pull/41)